dev_docs/kibana_platform_plugin_intro.mdx

```diff
@@ -69,19 +69,19 @@ We will continue to focus on adding clarity around these types of services and w
 ### Core services
 
 Sometimes referred to just as Core, Core services provide the most basic and fundamental tools neccessary for building a plugin, like creating saved objects,
-routing, application registration, and notifications. The Core platform is not a plugin itself, although
+routing, application registration, notifications and <DocLink id="kibCoreLogging" text="logging"/>. The Core platform is not a plugin itself, although
```

Comment:
AFAIK, this doc is only used in the new Docs system, so linking to the `kibCoreLogging` section should be ok.